### PR TITLE
pagination capability for PRs API

### PR DESCRIPTION
**Pull request by aman7870**
I did the changes for getting the selected page value from user

### CODE REVIEW

**Review by swarajpure — CHANGES_REQUESTED** [review]
Checks are failing

Please do `npm run lint-fix` to fix the errors and commit again

**Comment by aman7870** [comment]
fixed the build errors

**Review by swarajpure — CHANGES_REQUESTED** [review]
Please pass a default value of 1 in function call
Otherwise if the user simply calls `pullrequests/open` without `page` as query, a blank response will be returned.

**Comment by aman7870** [comment]
fixed the page number to be 1 by default

File: controllers/pullRequestsController.js [hunk 1]
```diff
@@ -107,7 +107,12 @@ const getStalePRs = async (req, res) => {
  */
 const getOpenPRs = async (req, res) => {
   try {
-    const { data } = await githubService.fetchOpenPRs()
+    let pageNumber = 1
+
+    if (req.query.page > 1) {
+      pageNumber = req.query.page
+    }
+    const { data } = await githubService.fetchOpenPRs(pageNumber)
```
Comment (swarajpure) [inline controllers/pullRequestsController.js:115]: There is a better and short way to pass default arguments

Please have a look here:
https://github.com/Real-Dev-Squad/website-backend/blob/d17ea626918d1424cc8ebd8181794c572e96ac24/models/users.js#L60

Also, please make sure to convert the argument to integer using `parseInt` as done in above function.

**Review by swarajpure — CHANGES_REQUESTED** [review]

**Comment by aman7870** [comment]
did the changes do check

File: controllers/pullRequestsController.js [hunk 2]
```diff
@@ -107,7 +107,8 @@ const getStalePRs = async (req, res) => {
  */
 const getOpenPRs = async (req, res) => {
   try {
-    const { data } = await githubService.fetchOpenPRs()
+    const pageNumber = ((req.query.page) || 1)
+    const { data } = await githubService.fetchOpenPRs(pageNumber)
```
Comment (swarajpure) [inline controllers/pullRequestsController.js:111]: Can we simply pass it to the function than saving it in variable and then passing it?
`fetchOpenPRs(req.query.page || 1)` would be short and sweet

**Review by swarajpure — CHANGES_REQUESTED** [review]
Just one final, simple change and we'll be able to merge it!

**Comment by aman7870** [comment]
did the changes do check

**Review by swarajpure — APPROVED** [review]
Great job!
Thank you! 🎉

**Pull request merged**